pkg/scheduler/api/cluster_info.go

```diff
@@ -26,8 +26,8 @@ type ClusterInfo struct {
 	Nodes          map[string]*NodeInfo
 	Queues         map[QueueID]*QueueInfo
 	NamespaceInfo  map[NamespaceName]*NamespaceInfo
-	RevocableNodes map[string]*NodeInfo
-	NodeList       []string
+	RevocableNodes map[string]*NodeInfo `json:"-"`
+	NodeList       []string             `json:"-"`
```

Comment:
why only change these two variables?

Reply:
The original PR is to support dump scheduler cache snapshot to json file. Those fields may not concerned by user,  so skip them.

Reply:
I think we just need fix extender marshal err in this pr, and the changes here can be done by original PR and make some comments.